packages/logger-winston/src/index.ts

```
@@ -0,0 +1,99 @@
+      if (typeof messageArg === 'function') {
+        messageArg = messageArg();
+      }
+      if (messageArg?.toJSON) {
+        messageArg = messageArg.toJSON();
+      }
+      if (messageArg instanceof AggregateError) {
+        return messageArg.errors;
+      }
+      return messageArg;
+    });
+  let message: string = '';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)